src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,55 @@
+package duke;
+
+import duke.command.Command;
+
+/**
+ * Duke is a Chatbot that is able to store and keep track of tasks inputted by the user.
+ */
+public class Duke {
+
+    private Storage storage;
+    private TaskList tasks;
+    private Ui ui;
+
+    /**
+     * Constructor for Duke.
+     *
+     * @param filePath The relative path of the location to store the tasks.
+     */
+    public Duke(String filePath) {
+        ui = new Ui();
+        storage = new Storage(filePath);
+        try {
+            tasks = new TaskList(storage.load());
+        } catch (Exception e) {
```

Comment:
I think it may not be a good practice to catch `Exception` as a "pokemon catch". Would you want to consider specifying the type of exception you want to handle?